messages/retrieve.start.md

```diff
@@ -191,7 +191,7 @@ The retrieve target directory [%s] overlaps one of your package directories. Spe
 
 # wantsToRetrieveCustomFields
 
-Because you're retrieving one or more CustomFields, we're also retrieving the CustomObject to which it's associated.
+Because you're retrieving one or more CustomFields that you didn't specify the name for, we're also retrieving the CustomObject to which it's associated.
```

Comment:
```suggestion
Because you're retrieving one or more CustomFields, and you didn't specify any field names, we're also retrieving the CustomObject to which they're associated.
```

Reply:
Okay?

Reply:
> and you didn't specify any field

nuance: you might have specified **some** field names, but you still have a wildcard on CustomField, too.

Reply:
So, final message after chatting:

Because you're retrieving one or more CustomFields that you didn't specify the name for, we're also retrieving the CustomObject to which they're associated.